schema.sql

```diff
@@ -0,0 +1,43 @@
+CREATE DATABASE comments;
+USE comments;
+
+CREATE TABLE comment (
+    id integer not null auto_increment,
+    text varchar(1000) not null,
```

Comment:
I tried looking this up more in detail - but for comments there can also be other types such as 'text, longtext, mediumtext'. Depending on what the length of comments are, there might be other options available.